### DISCUSSION TITLE
SeleniumBase SB Hide browser header

### DISCUSSION THREAD

**Discussion by ch-abbas** [seleniumbase/SeleniumBase#2419] [Q&A] [answered]
Can you please tell me by the use of SB lib in seleniumbase how can i able to hid opend borwser window url bar, tabs and all contorls. It will only show flat screen.

**Comment by mdmintz** [comment, marked as answer]
The only hiding option is `headless` mode:
Hide the entire browser window.

**Comment by OpsecGuy** [comment]
Use the maximized window option to see page content only.
